### ISSUE TITLE
Error using RxDart 0.21.0

### ISSUE THREAD

**Issue by chip2n** [felangel/bloc#104]
```
bloc.dart:24:44: Error: No named parameter with the name 'seedValue'.
```

This is most likely due to the following breaking change (from the changelog): 
> BehaviorSubject now has a separate factory constructor seeded()

**Comment by felangel** [comment]
@chip2n thanks for reporting this. Can you please include steps to reproduce this?

**Comment by yuyoDev** [comment]
Today I found the same mistake, and yesterday everything worked perfect,

bloc.dart:24:44: Error: No named parameter with the name 'seedValue'.
    _stateSubject = BehaviorSubject<State>(seedValue: initialState);

**Comment by wm17** [comment]
I've encountered the same issue. Any object that extends Bloc causes the issue, as the offending line is in the Bloc constructor (bloc.dart:24:44). It appears that `BehaviorSubject<State>(seedValue: initialState)` should now be `BehaviorSubject.seeded(initialState)` in rxdart 0.21.0 onwards.

**Comment by felangel** [comment]
#105 will address this. I'll let everyone know when I've published the change

**Comment by felangel** [comment]
Update: this is fixed in `bloc` `v0.9.3` so you should be able to just do a `flutter packages get` and the issue should be resolved.
